docs/5/api-reference/directives.md

```diff
@@ -249,12 +249,14 @@ directive @belongsToMany(
   """
   Allow clients to query paginated lists without specifying the amount of items.
   Overrules the `pagination.default_count` setting from `lighthouse.php`.
+  Setting this to `null` means clients have to explicitly ask for the count.
```

Comment:
```suggestion
  When a value is defined in `pagination.default_count` settings, setting this value to `null` in directive means clients have to explicitly ask for the count for this specific pagination.
```

Reply:
I don't think this longer explanation provides much value, the precedence is explained by the line above.